manifest.yml

```
@@ -0,0 +1,24 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"
+version: "202009.00"
+description: |- 
```

Why the hyphen `-` after the `|`?
```suggestion
description: |
```

---

Hyphen is needed here for multiple line sentence.